packages/trader/src/Modules/Trading/Containers/chart-widgets.tsx

```diff
@@ -37,6 +37,7 @@ export const DigitsWidget = observer(({ digits, tick }: { digits: TDigits['digit
             display_status={display_status}
             is_digit_contract={is_digit_contract}
             is_ended={is_ended}
+            is_mobile={is_mobile}
```

Comment:
Extract `is_mobile` in order to replace deprecated `isMobile()` in packages/trader/src/Modules/Contract/Components/Digits/digits.tsx. Also added sanity check for this case in the card